src/AppBundle/DataFixtures/ORM/LoadFixtures.php

```diff
@@ -34,8 +34,7 @@
  */
 class LoadFixtures implements FixtureInterface, ContainerAwareInterface
```

Comment:
no. The class is still container aware (but it uses the trait to implement it instead of doing it explicitly)